public/javascripts/privateSongForm.js

```diff
@@ -0,0 +1,219 @@
+var privateSongForm = function(song, type, readonly) {
+    var title = m.prop(song.title)
+    var author = m.prop(song.author)
+    var lang = m.prop(song.lang === '' ? 'english' : song.lang)
```

Comment:
hmm...I feel like the default language should be loaded differently than hardcoded.